### PR TITLE
fix(toast): update toast design to match MD spec

### PR DESCRIPTION
**Pull request by brandyscarney**
#### Short description of what this resolves:
Updates the Material Design Toast to closer match the spec: https://material.io/design/components/snackbars.html

#### Changes proposed in this pull request:

- Updates the animation to use opacity, not translate
- Updates design with the right background, box-shadow, etc.
- Fixes the broken position middle of toast (https://github.com/ionic-team/ionic/issues/16271) and updates e2e test to include this
- Allows for line breaks to be passed in the message

**Ionic Version**: 4.x

**Fixes**: #16271

| `master` | `fix-toast-design` |
| ----------| -------------------|
| localhost_3333_src_components_toast_test_basic galaxy s5 5 | localhost_3333_src_components_toast_test_basic galaxy s5 2 |

### CODE REVIEW

File: core/src/components/toast/toast.tsx [hunk 1]
```diff
@@ -28,9 +28,12 @@ export class Toast implements ComponentInterface, OverlayInterface {
   animation: Animation | undefined;
 
   @Prop({ connect: 'ion-animation-controller' }) animationCtrl!: HTMLIonAnimationControllerElement;
+
   @Prop({ context: 'config' }) config!: Config;
 
-  /** @internal */
+  /**
```
Comment (manucorporat) [inline core/src/components/toast/toast.tsx]: I was actually moving to:
```
/** @internal */
```
thoughts?

File: core/src/components/toast/toast.tsx [hunk 2]
```diff
@@ -201,7 +204,7 @@ export class Toast implements ComponentInterface, OverlayInterface {
       <div class={wrapperClass}>
         <div class="toast-container">
           {this.message !== undefined &&
-            <div class="toast-message">{this.message}</div>
+            <div class="toast-message" innerHTML={this.message}></div>
```
Comment (manucorporat) [inline core/src/components/toast/toast.tsx]: security, can we make this change in a different PR?

File: core/src/components/toast/toast.tsx (hunk 1 above)
Comment (brandyscarney) [reply core/src/components/toast/toast.tsx]: I'd just prefer if it were in the same format as the other comments, idk

**Comment by brandyscarney** [comment]
Good to merge once checks pass 👍
